code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -737,6 +740,9 @@ What are the archived variables for?
 	if(total_heat_capacity > 0)
 		temperature = total_thermal_energy/total_heat_capacity
 
+	if(temperature <= 0 || isnan(temperature) || !isnum(temperature))
+		CRASH("A pipenet with [length(mixtures)] connected airs is corrupt and cannot flow safely. Pipenet root is [root] at ([root.x], [root.y], [root.z]).")
```

Comment:
can we make this error message slightly different? so that we can differentiate what is wrong?